plugins/polls/lib/foodsoft_polls/engine.rb

```diff
@@ -0,0 +1,15 @@
+module FoodsoftPolls
+  class Engine < ::Rails::Engine
+    def navigation(primary, context)
+      return unless FoodsoftPolls.enabled?
+      return if primary[:foodcoop].nil?
+      sub_nav = primary[:foodcoop].sub_navigation
+      sub_nav.items <<
+        SimpleNavigation::Item.new(primary, :polls, I18n.t('navigation.polls'), context.polls_path)
+      # move to right before tasks item
+      if i = sub_nav.items.index(sub_nav[:tasks])
+        sub_nav.items.insert(i, sub_nav.items.delete_at(-1))
+      end
```

Comment:
Perhaps a fallback for when the tasks item is not found. You can get the index of the tasks item and add it right before, and if it isn't found, add it at the end. Another plugin may disable the tasks menu item.

Reply:
the `if` does exactly that: if `:tasks` has been found will be removed and inserted at the corresponding position

Reply:
Ah, now I see, super :)

Reply:
Ah, I missed that before, super.